erpnext/hr/doctype/employee/employee.js

```diff
@@ -17,6 +17,11 @@ erpnext.hr.EmployeeController = frappe.ui.form.Controller.extend({
 	refresh: function() {
 		var me = this;
 		erpnext.toggle_naming_series();
+		frappe.ui.form.on("Employee External Work History", {
```

Comment:
don't bind on `refresh` it will keep binding again and again. This should be at the top level